### PR TITLE
Fx naming conflicts

### PR DESCRIPTION
**Pull request by CYIMANA-Faisal**
# Description

## Type of change

Please check the options that are relevant.

- [☑︎] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How should this be manually tested or reproduced?

Please all you have is to run your server locally and refer to the screenshot bellow to get the endpoint link

# Checklist:

- [☑︎] My code follows the style guidelines of this project
- [☑︎] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [☑︎] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots (if applicable)

Screen Shot 2021-01-19 at 06 53 51
Screen Shot 2021-01-19 at 06 54 12
Screen Shot 2021-01-19 at 06 54 33
Screen Shot 2021-01-19 at 06 55 19
Screen Shot 2021-01-19 at 06 55 27

### CODE REVIEW

File: src/migrations/20201020143658-create-trip.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ module.exports = {
       returnDate: {
         type: Sequelize.DATE
       },
-      AccommodationId: {
+      accommodationId: {
```
Comment (bdushimi) [inline src/migrations/20201020143658-create-trip.js:26]: @CYIMANA-Faisal create a new migration file that changes the name of this column

File: src/migrations/20201109053127-create-amenity.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.UUID
       },
-      AccommodationId: {
+      accommodationId: {
```
Comment (bdushimi) [inline src/migrations/20201109053127-create-amenity.js:10]: @CYIMANA-Faisal create a new migration file that changes the name of this column

File: src/migrations/20201123045517-create-bookings.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ module.exports = {
       username: {
         type: Sequelize.STRING,
       },
-      AccommodationId: {
+      accommodationId: {
```
Comment (bdushimi) [inline src/migrations/20201123045517-create-bookings.js:13]: @CYIMANA-Faisal create a new migration file that changes name of this column

File: src/migrations/20201123125725-create-reviews.js [hunk 4]
```diff
@@ -15,7 +15,7 @@ module.exports = {
           key:'id'
         }
       },
-      AccommodationId: {
+      accommodationId: {
```
Comment (bdushimi) [inline src/migrations/20201123125725-create-reviews.js:18]: @CYIMANA-Faisal create a new migration file that changes the name of this column

**Review by bdushimi — CHANGES_REQUESTED** [review]

**Review by songa1 — COMMENTED** [review]
I think they are good, except for a few changes that Bernard suggested, and I'm still getting an error saying that `original: error: column "AccommodationId" does not exist` even in this branch.

**Pull request closed without merging**